### DISCUSSION TITLE
change WiFi settings remotely

### DISCUSSION THREAD

**Discussion by Johnwulp** [jomjol/AI-on-the-edge-device#762] [Q&A] [answered]
Hi,

Is it possible to update the WiFi settings remotely? I have this esp thing already a long time, for some reason I don’t have the required sd-card that contains the ini file with these settings. In the file browser I do see an “_wlan.ini” file, but that’s binary. 

Thanks in advance!

**Comment by jomjol** [comment]
No you cannot change the wlan configuration remotely. See here: https://github.com/jomjol/AI-on-the-edge-device/wiki/Installation#wlan

**Reply by Johnwulp** [reply]
I can’t remember it exactly, but how does one configure wlan settings without an sd-card? I don’t have one. Is it flashed into the esp module itself?

**Comment by jomjol** [comment, marked as answer]
Without sd card this system will not work. The sd card is needed in any case.

**Reply by Johnwulp** [reply]
You are right! I took the module apart, and there is an SD card on it. Reconfigured it and works :-)

Thanks!
